certified-connectors/Tesseron Invoice/readme.md

```diff
@@ -0,0 +1,41 @@
+## Tesseron ASM Ticket Connector
+With Tesseron, we offer your company an innovative solution for cooperation with your customers, partners and suppliers.This connector allows you to use Tesseron Invoice API to create position notes and activity recordings.
+
+## Publisher: Publisher's Name
+Lutihle + Luithle GmbH
+
+
+## Prerequisites
+You will need the following to proceed:
+* A Tesseron Instance
+* A Tesseron licensed user
+* An API Key (Service: Invoice)
+
+
+## Supported Operations
+The connector supports the following operations:
+* `Create Inovice Position Note`: Create a position note with or without tickets
```

Comment:
@developmentLL all files should be free of grammatical errors and typos. here you have "Create Inovice...", can you please change to "Create Invoice...".